test/e2e/global_ca_test.go

```diff
@@ -32,6 +32,14 @@ import (
 )
 
 func TestGlobalCA(t *testing.T) {
+
+	// Skip if it is the resilience pipeline because the ChaosJob can prevent
+	// assert_operator_has_been_restarted_once_more to pass when it deletes an operator Pod
+	// exactly on restart.
+	if test.Ctx().Pipeline == "e2e/resilience" {
+		t.Skip()
+	}
```

Comment:
I am OK with this as a temporary fix. But maybe we should make the assertion more robust.

Reply:
Ack, I leave #6356 open.